commonmark/src/main/java/org/commonmark/html/HtmlRenderer.java

```diff
@@ -46,6 +50,12 @@ public void render(Node node, Appendable output) {
         RendererVisitor rendererVisitor = new RendererVisitor(new HtmlWriter(output), customHtmlRenderers);
         node.accept(rendererVisitor);
     }
+    
+    public void render(Node node, OutputStream output) throws IOException {
+        Writer writer = new OutputStreamWriter(output);
```

Comment:
This uses the default character encoding, which is almost never what you actually want (except on Linux, where it's UTF-8).

On second thought, I think the existing method with `Appendable` is enough; `Writer` also implements that, so the functionality is already there. Sorry that this has caused you to do unnecessary work :(. I'll close #3.